src/workloads/c2c/ChangeEventApplication.yml

```diff
@@ -13,6 +13,9 @@ Keywords:
 - change event application
 - CEA
 
+EnvironmentVariables:
+  MongosyncConnectionUrls: http://localhost:27182
+
```

Comment:
Instead of a global thing should this be part of the ExternalScriptRunner actor config? It really only seems relevant to that actor, and having something called "EnvironmentVariables" inside the workload context might make future actor authors think it's a general scheme for reading whatever environment variables genny can see

Reply:
Is there a way to override an actor specific config with `overrides.yml`? I was using that to pass in the connection string here.

Alternatively, I could just invoke genny with the environment variables required (e.g. something like
`MongosyncConnectionUrls="A B C" ./run-genny workload ...`). I think the child processes should inherit the environment variables.

Reply:
The `overrides.yml` file can override any key/value pair in the Genny config. So in theory you could override the specific actor config, though you would have to specify the paths to it inside the override yml, meaning it would be brittle for changing actor names / numbers I believe.

I still think the name `EnvironmentVariables` would be confusing though. Maybe there could be a global actor shared space? Like renaming it to `ActorVars` with a subkey for `ExternalScriptRunner` or something?

cc @rtimmons since this relates to the slack convo about config schemes

Reply:
The `ActorVars` with a subkey for `ExternalScriptRunner` sounds good to me. Then the fact that the actor implements access to the global actor space in terms of env variables is hidden and easier to change in the future & other actors could implement it differently. Cool if I proceed with that option?

Reply:
Using env vars for conn strings can be dangerous if the conn strings have any auth in them (a proc's env vars aren't private). Plus we don't have precedent for plumbing with env vars, so this could end up rather confusing. I think we can do something better within DSI to expose configdict in a sane way.